internal/util/remount.go

```diff
@@ -0,0 +1,123 @@
+package ebutil
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/prometheus/procfs"
+)
+
+// TempRemount iterates through all read-only mounted filesystems, bind-mounts them at dest,
```

Comment:
Is there a strong point why we should keep it in `internal`?

Reply:
Only reason is so that it doesn't get imported by anyone else.

Reply:
I'm always in favor of keeping things open, but I won't strongly oppose here :)